docs-src/concepts/what-is-visibility.md

```diff
@@ -5,6 +5,13 @@ sidebar_label: Visibility
 description: The term Visibility, within the Temporal Platform, refers to the subsystems and APIs that enable an operator to view Workflow Executions that currently exist within a Cluster.
 tags:
   - term
+ssdi:
+  - For Temporal Server v1.19 and earlier, all supported databases for Visibility provide standard Visibility features, and an Elasticsearch database is required for advanced Visibility features.
+  - For Temporal Server v1.20 and later, advanced Visibility features are enabled on all supported SQL databases, in addition to Elasticsearch.
+  - From Temporal Server v1.21 and later,
```

Comment:
```suggestion
  - In Temporal Server v1.21 and later:
  
```
The second-level list didn't render correctly, so the blank line is one possibility to try. It's possible that the ssdi element doesn't support nested lists.

Reply:
will check on this